From 7eaa810935ff190e039f465a06663b4d92aea794 Mon Sep 17 00:00:00 2001
From: Hannes Reinecke <hare@suse.de>
Date: Wed, 4 Mar 2015 16:32:17 +0100
Subject: [PATCH] Allow up to 4096 simultaneous connections

On large system we hit the limit on 512 simultaneous dbus
connections, resulting in tons of annoying messages:

Too many concurrent connections, refusing

This patch raises the limit to 4096.

(cherry picked from commit cbecf9bf929318533fea798c57c10efcf6b2b447)
---
 src/core/dbus.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/dbus.c b/src/core/dbus.c
index 260775cd85..85b5174868 100644
--- a/src/core/dbus.c
+++ b/src/core/dbus.c
@@ -44,7 +44,7 @@
 #include "bus-internal.h"
 #include "selinux-access.h"
 
-#define CONNECTIONS_MAX 512
+#define CONNECTIONS_MAX 4096
 
 static void destroy_bus(Manager *m, sd_bus **bus);
 
